src/features/events/components/EventTypeAutocomplete.tsx

```diff
@@ -250,6 +253,7 @@ const EventTypeAutocomplete: FC<EventTypeAutocompleteProps> = ({
                         },
                         warningText: messages.type.deleteMessage({
                           eventType: option.title,
+                          orgTitle: orgTitle ? orgTitle : currentDefaultOrgMsg,
```

Comment:
Under no normal circumstances should `orgTitle` be `undefined` on this page. If it is, it should be temporary (while loading) and I'm thinking we can fallback to an empty string or something. That way, we don't need to have two separate messages.